### PR TITLE
Fix missing fblike button in blog

### LINKED ISSUE #814: Bring back the facebook social button on blog posts
### Is this a bug report?

No. It was just removed.

### Have you read the [Contributing Guidelines on issues](https://github.com/facebook/Docusaurus/blob/master/CONTRIBUTING.md#reporting-new-issues)?

Yes

### Environment

Docusaurus 1.3.0

### Steps to Reproduce

Go to https://docusaurus.io/blog/2018/04/30/How-I-Converted-Profilo-To-Docusaurus

### Expected Behavior

The button would be there.

screenshot 2018-06-29 13 39 45

### Actual Behavior

No button.

### Reproducible Demo

N/A

### PR DESCRIPTION
**Pull request by endiliey**
## Motivation

Fix #814 

It should be
```js
className="fb-like"
```

According to https://developers.facebook.com/docs/plugins/like-button/

Instead of
```js
className="fbLike"
```

### Have you read the [Contributing Guidelines on pull requests](https://github.com/facebook/Docusaurus/blob/master/CONTRIBUTING.md#pull-requests)?

Yes

## Test Plan

asd

## Related PRs

#757 renamed "fb-like" to "fbLike"

### CODE REVIEW

File: lib/static/css/main.css [hunk 1]
```diff
@@ -2050,7 +2050,7 @@ input::placeholder {
   padding-bottom: 5px;
 }
 
-.fbLike {
+.fb-like {
```
Comment (yangshun) [inline lib/static/css/main.css]: Could you add a comment here that we shouldn't change this class because it's determined by FB SDK?

Comment (endiliey) [reply lib/static/css/main.css]: I think it should be in the js ? the css here just add extra style

Comment (yangshun) [reply lib/static/css/main.css]: Yes but I renamed it thinking it was our own styling 😥

File: lib/core/BlogPostLayout.js [hunk 2]
```diff
@@ -24,7 +24,7 @@ class BlogPostLayout extends React.Component {
       this.props.config.facebookComments && (
         <div className="blogSocialSectionItem">
           <div
-            className="fb-comments"
+            className="fb-comments" // Facebook SDK require it to be 'fb-comments'
```
Comment (yangshun) [inline lib/core/BlogPostLayout.js]: Please make sure this doesn't show up in the output. I don't remember being able to write comments in JSX like that. I had to do 

```
{ /* Facebook SDK require it to be 'fb-comments */ }
```

File: lib/static/css/main.css (hunk 1 above)
Comment (yangshun) [reply lib/static/css/main.css]: Ah sure, let's write it in the JS then.

File: lib/core/BlogPostLayout.js (hunk 2 above)
Comment (endiliey) [reply lib/core/BlogPostLayout.js]: ah that's true. I almost forgot 😢

**Comment by JoelMarcey** [comment]
@endiliey @yangshun Thanks a lot for working to fix this. I am glad it is back 👍 I guess we will just have to live with the fact that we lost all of our likes for the two Docusaurus blog posts? 😢

**Comment by yangshun** [comment]
I don't think the likes will be lost. Likes don't work that way 🤔 

If I'm not wrong, I think the like button paths were not correctly configured in the past, and they were not post-specific. The buttons on all the posts were pointing to the same path.

**Comment by JoelMarcey** [comment]
I think the like buttons were pointing to a Dev version maybe. Maybe I am the only one in the world who saw hundreds of likes on the announcement blog post 🤣 

<sub>Sent with <a href="http://githawk.com">GitHawk</a></sub>

**Comment by yangshun** [comment]
Nope, I saw it too (942 likes). Actually, I just saw it 5 mins ago while fiddling with the config trying to find the right config for it. But I changed something and am unable to get back that state. Good news is it's definitely there, bad news is I need more time to find it back 😞

**Comment by yangshun** [comment]
I got it. It had to be non-HTTPS URL and using the `index.html` version.

screen shot 2018-07-01 at 10 29 38 pm

screen shot 2018-07-01 at 10 31 47 pm

Play around with the URL http://docusaurus.io/blog/2017/12/14/introducing-docusaurus.html on https://developers.facebook.com/docs/plugins/like-button/

Now that we have changed to clean URLs we probably can't get back to that state too 😥 

cc @endiliey

**Comment by endiliey** [comment]
We can point to the html version if we want even in clean url setting.
But if its only for http then we won't be able to get it back

**Comment by yangshun** [comment]
> We can point to the html version if we want even in clean url setting.

It'll affect end users unless we add another config? But then again, not too keen on adding more configs.

> But if its only for http then we won't be able to get it back

Weird though, not sure how the HTTP version got so many likes when we were on HTTPS the whole time.

**Comment by endiliey** [comment]
> It'll affect end users unless we add another config? But then again, not too keen on adding more configs.

Hmm lets leave it then.

> Weird though, not sure how the HTTP version got so many likes when we were on HTTPS the whole time.

I think when they first launched it's still on http, just like when jest first changed to jestjs.io its still http

Or .. it points to the wrong one before

**Pull request merged**